README.md

```diff
@@ -7,46 +7,17 @@ It is supposed to:
 + Talk to TiKV for reading/writing data and the resulted data is encoded/decoded just like what we do in TiDB.
 + Talk to Coprocessor for calculation pushdown
 
-## How to build
+## Quick Start
```

Comment:
We may rename this section to `Example`?

Reply:
I think quick is better than examples, it indicates that this is a quick task to get familiar with java client.